src/Evolution/DiscontinuousGalerkin/MortarTags.hpp

```diff
@@ -93,4 +94,26 @@ struct BoundaryMessageFromInbox : db::SimpleTag {
   using type = std::unordered_map<Key, std::unique_ptr<BoundaryMessage<Dim>>,
                                   boost::hash<Key>>;
 };
+
+/// Tag to hold the subcell neighbor data that was constructed in
+/// ComputeTimeDerivative.
+///
+/// This tag differs from
+/// `evolution::dg::subcell::Tags::NeighborDataForReconstruction` because this
+/// tag
+/// 1. Only holds (in addition to the actual data) info about the Direction
+/// 2. Is a vector of DirectionMap%s for each buffer so we can have multiple
+/// buffers
+/// 3. Keeps track of which buffer we are using in the `size_t` of the pair.
+///
+/// This tag is not in the subcell namespace because it must be included in all
+/// executables, even if they don't use subcell. And there's nothing explicitly
+/// in this tag that has to do with subcell so we keep it here in just the
+/// `evolution::dg::Tags` namespace.
```

Comment:
If it's subcell-specific what is it used for when there's no subcell?  If it's not, why does it have "Subcell" in the name?

Reply:
If we aren't doing subcell, this tag never gets used. If we are doing subcell, then it gets used. I guess I could move this tag into the subcell namespace in `DgSubcell/Tags/NeighborData.hpp` , forward declare it in ComputeTimeDerivative, and then only use it inside `if constexpr (using_subcell_v)`. This avoids having the DG/Evolution library depend on the subcell library.